<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorUI</class>
 <widget class="QMainWindow" name="MonitorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>161</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Server Monitor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #e8edea</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>106</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Server Info</string>
    </property>
    <widget class="QLabel" name="label_ip">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_port">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>66</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ss">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ipv">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>39</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>http://127.0.0.1/</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_portv">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>74</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>8888</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_blue">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/resource/blue.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_black">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>95</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/resource/black.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="autoRepeatDelay">
     <number>306</number>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_all">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>147</y>
      <width>81</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-image: url(:/resource/ip_normal.png);
    color: #ffffff;
}
QPushButton:pressed {
    border-image: url(:/resource/ip_pressed.png);
    color: #ffffff;
}

</string>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
